installers/msi-language/GAPixel/GAPixelCA.cs

```diff
@@ -74,6 +75,11 @@ protected override WebRequest GetWebRequest(Uri uri)
         public static ActionResult SendPixel(Session session)
         {
             var wv = System.Environment.OSVersion.VersionString;
+            if (System.Environment.OSVersion.Version.Major > 6)
+            {
+                session.Log("skipping GA tracking for Windows > 8.1 version is: {0} major version: {1}", wv, System.Environment.OSVersion.Version.Major);
+                return ActionResult.Success;
+            }
```

Comment:
```suggestion
```

If we're already in the custom action we might as well fire it. The concern was not having any delays on Windows 10 which I now realize is moot anyway as it wouldn't be firing the event anyway.